finetuner/tuner/base.py

```diff
@@ -92,6 +92,9 @@ def __init__(
         self._scheduler = None
         self._device_name = device
 
+        # Check for early stopping
+        self.stop_training = False
```

Comment:
I would make this a private property, like all others

```suggestion
        self._stop_training = False
```

Reply:
Bo specifically asked for this to be public in an above comment, originally it was private, I think @bwanglzu  has some use in mind for this in other modules

Reply:
Got it, let's have it public then